Sources/Extensions/NSPredicate+PrediKit.swift

```diff
@@ -19,7 +19,7 @@ public extension NSPredicate {
      - type: The `Reflectable` class type that you'll be querying against. The type you supply here is what PrediKit will inspect to ensure the property names you specify in your includers are contained in that class' property list.
      - builder: A closure that you use to generate includers that construct each subpredicate in the created `NSPredicate`
      */
-    convenience init<T: Reflectable>(_ type: T.Type, builder: ((_ includeIf: PredicateBuilder<T>) -> Void)) {
```

Comment:
Why are we removing `Reflectable` here? What was the problem it caused? If I remember correctly, I   put this here to enforce the developer to ensure that whatever type they feed into this conforms to `Reflectable`. The only class conforming to it is `NSObject` at the moment since this library was intended for use with CoreData. Is there a case where you can use PrediKit with straight up Swift-only objects? Is there a particular API where that's needed?

Reply:
The same was here ```NSPredicate+PrediKit.swift:22:25: Redundant conformance constraint 'T': 'Reflectable'```

Reply:
Good to know! Makes sense now too